presto-docs/src/main/sphinx/release/release-340.md

```diff
@@ -0,0 +1,28 @@
+# Release 339 (XX Jul 2020)
+
+## General Changes
+
+* Add support for query parameters in `LIMIT`, `OFFSET` and `FETCH FIRST` clauses. ({issue}`4529`, {issue}`4601`)
+* Improve dynamic partition pruning for short queries. ({issue}`4388`}
+
+## Hive Connector Changes
+
+* Improve performance when querying tables with `skip.header.line.count` property set to 1. ({issue}`4513`)
```

Comment:
Should we mention that this only affects textfiles that spans across several splits?

Reply:
like
```
... when querying a table with large files and with `skip.header.line.count` property set to 1.
```

?

Reply:
👍